redis/redis-connector/redis-connector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/Append.java

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Why custom mediator is used here. can do same as in [1]

[1] https://github.com/wso2/esb-connectors/blob/master/salesforce/salesforce-connector/salesforce-connector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/salesforce/SetupCreateSobjects.java